articles/quickstart/backend/golang/01-authorization.md

```diff
@@ -199,15 +199,24 @@ type CustomClaims struct {
 }
 
 func checkScope(scope string, tokenString string) bool {
-	token, _ := jwt.ParseWithClaims(tokenString, &CustomClaims{}, nil)
-
-	claims, _ := token.Claims.(*CustomClaims)
+	token, _ := jwt.ParseWithClaims(tokenString, &CustomClaims{}, func (token *jwt.Token) (interface{}, error) {
+		cert, err := getPermCert(token)
```

Comment:
This line should be:
`cert, err := getPemCert(token)`

Reply:
```suggestion
		cert, err := getPemCert(token)
```